src/config.ts

```diff
@@ -2,14 +2,12 @@ import * as fs from 'fs'
 import { fileURLToPath } from 'url'
 import { dirname, join } from 'path'
 import JSON5 from 'json5'
-import { pino } from './utils/logger.js'
 
 const currDir = dirname(fileURLToPath(import.meta.url))
 const rootDir = join(currDir, '..')
 
 const configName = process.argv.slice(2)[0] || 'default'
 const configFileName = `config.${configName}.json5`
-pino.logger.debug('Using config file:', configFileName)
```

Comment:
Why you removed logging here? Just use `logger.info()` instead of `logger.debug()`

Reply:
@bludnic Because to configure the logger, i need to connect the config to the logger. Therefore, logging in the config file leads to an error.

Reply:
@PaulDremanovich Can you then log that message in other place, e.g. in `index.ts` file?

Reply:
@bludnic If it is very important to log this information, then I could export the name of the configuration file and log it, for example, in the index.ts. What do you think ?